Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (0.9.2)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
     mini_mime (1.1.5)
     mini_portile2 (2.8.5)
     minitest (5.11.3)
-    minitest-rails (6.1.0)
+    minitest-rails (6.1.1)
       minitest (~> 5.10)
       railties (~> 6.1.0)
     mixlib-shellout (3.2.7)
       chef-utils
-    mocha (1.2.1)
-      metaclass (~> 0.0.1)
+    mocha (1.16.1)
     momentjs-rails (2.29.4.1)
       railties (>= 3.1)
-    msgpack (1.4.2)
+    msgpack (1.7.2)
     multi_json (1.15.0)
     multi_xml (0.6.0)
     multipart-post (2.3.0)
-    mysql2 (0.5.3)
+    mysql2 (0.5.5)
```

Comment:
### [mysql2](https://github.com/brianmario/mysql2)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.3 → 0.5.5

[_[change-log](https://github.com/brianmario/mysql2/releases/tag/0.5.5), [source-code](https://github.com/brianmario/mysql2/tree/0.5.5), [gem-diff](https://my.diffend.io/gems/mysql2/0.5.3/0.5.5)_]





<details>
<summary>Commits</summary>

A change of **112** commits. See the full changes on [the compare page](https://github.com/brianmario/mysql2/compare/0.5.3...0.5.5).

These are the first 10 commits:
- (785969f) [The taint mechanism will be deprecated in Ruby 2.7 (#1083)](https://github.com/brianmario/mysql2/commit/785969fbceadd96d495ec9fecc67efee03d0056c)
- (f62edc0) [Made argument types strict for ruby 2.7 (#1096)](https://github.com/brianmario/mysql2/commit/f62edc0d17b3c31fdf2d5581a82feec8393239d2)
- (475c450) [Update rake (and rake-compiler) to suppress the warning. (#1…](https://github.com/brianmario/mysql2/commit/475c450c7698aab76b26e7d12dd273fb5f5948a9)
- (d25db07) [For local debugging, irb is Gemified since Ruby 2.6](https://github.com/brianmario/mysql2/commit/d25db07e25b9974981b86938c00206c4380618c2)
- (b52f27e) [Travis CI add Ruby 2.7 to the matrix](https://github.com/brianmario/mysql2/commit/b52f27e7cafcc264eff99a048ef3b881fb9f2888)
- (f8560c5) [Fix crash if a Mysql2::Client object is allocated but never …](https://github.com/brianmario/mysql2/commit/f8560c551bf1999baf7df43290e8f89471e77af4)
- (982dbdb) [Remove Enumerable include from Statement class (#1104)](https://github.com/brianmario/mysql2/commit/982dbdb1d521c668547cfd4ab927558c689b8bfd)
- (1b15d78) [Add project metadata to the gemspec (#1089)](https://github.com/brianmario/mysql2/commit/1b15d78a113b5c4f89ecd00e82ac40c63b8c7b55)
- (48172b5) [Don't call mysql_close if mysql_init wasn't called. (#1111)](https://github.com/brianmario/mysql2/commit/48172b579343a8c2cd3a4306ae110e06518d1039)
- (cb9e941) [Client session tracking (#1092)](https://github.com/brianmario/mysql2/commit/cb9e9411a885ece422a73508876d601fda6fae40)


</details>